geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisKey.java

```diff
@@ -92,15 +73,15 @@ public int getDSFID() {
 
   @Override
   public void toData(DataOutput out, SerializationContext context) throws IOException {
-    out.writeShort(crc16);
+    out.writeShort(slot);
     DataSerializer.writeByteArray(value, out);
   }
 
   @Override
   public void fromData(DataInput in, DeserializationContext context)
       throws IOException {
     // Need to convert a signed short to unsigned
-    crc16 = in.readShort() & 0xffff;
+    slot = in.readShort() & 0xffff;
```

Comment:
since slot will never be negative I think you can drop the "& 0xffff".

Reply:
done